dm/master/config.go

```diff
@@ -439,17 +439,21 @@ func parseURLs(s string) ([]url.URL, error) {
 	return urls, nil
 }
 
-func genEmbedEtcdConfigWithLogger() *embed.Config {
+func genEmbedEtcdConfigWithLogger(logLevel string) *embed.Config {
 	cfg := embed.NewConfig()
 	cfg.EnableGRPCGateway = true // enable gRPC gateway for the internal etcd.
 
 	// use zap as the logger for embed etcd
 	// NOTE: `genEmbedEtcdConfig` can only be called after logger initialized.
 	// NOTE: if using zap logger for etcd, must build it before any concurrent gRPC calls,
 	// otherwise, DATA RACE occur in NewZapCoreLoggerBuilder and gRPC.
-	// NOTE: we can only increase the log level for the clone logger but not decrease.
-	logger := log.L().WithFields(zap.String("component", "embed etcd")).WithOptions(zap.IncreaseLevel(zap.ErrorLevel))
-	cfg.ZapLoggerBuilder = embed.NewZapCoreLoggerBuilder(logger, logger.Core(), log.Props().Syncer) // use global app props.
+	logger := log.L().WithFields(zap.String("component", "embed etcd"))
+	// if logLevel is info, set etcd log level to WARN to reduce log
+	if strings.ToLower(logLevel) == "info" {
+		logger.Logger = logger.WithOptions(zap.IncreaseLevel(zap.WarnLevel))
```

Comment:
let's log this change to tell user. and "debug" maybe a level lower than "info"

Reply:
add log in 6d12af4
user can use debug level to see more log about etcd.